interview-questions.md

```diff
@@ -8,50 +8,74 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-  Your answer:
+  Your answer: 
+
+  I would describe a parameter as a variable that we can define in the function declaration. Parameters act kind of like place holders for a function that will later give it value. 
+
+  An argument is what passes value to parameters as mentioned earlier. When we pass value to a function we are usually calling it. I would say an argument is when you are trying to pass value or an expression to a function.
+
+ex: function favortieDessertflavors: (cookie, cake){
+  return chocolate, vanilla
+}
+
 
   Researched answer:
 
+A parameter is the variable notated inside the parenthsis that acts as a placeholder for what is expected to be called back in the function or given value. 
+
+An argument is the value or expression that is sent back to the parameter to run through the function. 
 
 
 2. The JavaScript built in method .map() takes predefined parameters. What are they? Which are required and which are optional?
 
   Your answer:
 
-  Researched answer:
+The .map() feature is a feature that creates a new array after going through each element in an array after a function has been called. The .map() feature uses the function command that was addressed to go through each element in the array one time and returns a new array with the new values that resulted. 
 
 
+  Researched answer:
+
+The .map() returns a new array with new values that resulted from going through an arrays indivual elements.
```

Comment:
The parameters mentioned here are currentValue, index and array.  The required parameter is currentValue (which is a placeholder name and could be called value or v).  Ex: `something.map(value => value * 5`